### PR TITLE
added search page

### PR DESCRIPTION
**Pull request by ishitakeshawani**
Preview link: https://deploy-preview-6--stunnershub.netlify.app/pages/search/search

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,152 +1,115 @@
...
-          class="link-no-style nav-link nav-icon-link"
-        >
-          <i class="fas fa-cart-shopping hide-icon"><span class="number-badge">0</span></i>
-          <span class="small-fontsize">Cart</span>
-        </a>
+<body>
+  <nav class="navbar semibold-font-weight">
+    <div class="openMenu"><i class="fa fa-bars"></i></div>
+    <a class="nav-icon-link nav-link link-no-style hide-icon" href="">
+      <img class="logoimage" src="/assets/Stunners.png" alt="" />
```
Comment (Naveenchandar) [inline index.html]: Please provide alt description for accessibility

File: index.html [hunk 2]
```diff
@@ -1,152 +1,115 @@
...
-        <img class="top-brands-item" src="/assets/brand3.jpg" alt="" />
-        <img class="top-brands-item" src="/assets/brand4.jpg" alt="" />
-      </div>
+  <div class="top-brands">
+    <div class="top-brands-title">Top Brands</div>
+    <div class="top-brands-list">
+      <img class="top-brands-item" src="/assets/brand1.jpg" alt="" />
+      <img class="top-brands-item" src="/assets/brand2.jpg" alt="" />
+      <img class="top-brands-item" src="/assets/brand3.jpg" alt="" />
+      <img class="top-brands-item" src="/assets/brand4.jpg" alt="" />
```
Comment (Naveenchandar) [inline index.html]: Please provide alt description for accessibility

File: pages/home/mobile-menu.js [hunk 3]
```diff
@@ -12,4 +14,9 @@ function show(){
 
 function close(){
     mainMenu.style.top = '-100%';
+}
+
+function searchItem(){
+    console.log("hugf");
```
Comment (Naveenchandar) [inline pages/home/mobile-menu.js]: You can remove the console

File: pages/Search/search.html [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+                    <i class="fa fa-times"></i>
+                </div>
+            </div>
+        </nav>
+        <div class="search-page">
+            <div class="search-result">Search results for "Liquid Lipstick" - 1 items</div>
+            <div class="cards-list">
+                <div class="card product-card">
+                    <div class="card-padding">
+                        <img class="product-img" src="/assets/product1.jpg" alt="" />
```
Comment (Naveenchandar) [inline pages/Search/search.html]: Please provide alt description for accessibility

File: pages/Search/search.html [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="author" content="Ishita Keshawani" />
+    <meta name="description" content="Search page of Stunners Hub" />
+    <meta name="keywords" content="beauty store,e-commerce,react.js" />
+    <link rel="stylesheet" href="https://felizui.netlify.app/styles/index.css" />
+    <link rel="stylesheet" href="/pages/home/homepage.css" />
+    <link rel="stylesheet" href="/pages/products/products.css" />
+    <link rel="stylesheet" href="/pages/Search/search.css">
+    <script src="https://kit.fontawesome.com/7744bfb8db.js" crossorigin="anonymous"></script>
```
Comment (Naveenchandar) [inline pages/Search/search.html]: Why scripts added for font awesome ?

File: pages/Search/search.html (hunk 4 above)
Comment (ishitakeshawani) [reply pages/Search/search.html]: done

File: index.html (hunk 2 above)
Comment (ishitakeshawani) [reply index.html]: done

File: pages/home/mobile-menu.js (hunk 3 above)
Comment (ishitakeshawani) [reply pages/home/mobile-menu.js]: yes

File: pages/Search/search.html (hunk 5 above)
Comment (ishitakeshawani) [reply pages/Search/search.html]: Added the wrong one by mistake.
Thank you for reviewing.

**Review by rohanmathur91 — APPROVED** [review]
Looks good to me

**Pull request merged**